nestkernel/growth_curve.h

```diff
@@ -227,8 +227,12 @@ class GrowthCurveGaussian : public GrowthCurve
   update( double t, double t_minus, double Ca_minus, double z, double tau_Ca, double growth_rate ) const override;
 
 private:
+  void compute_local();
```

Comment:
Private attributes should have names ending with an underscore:
```suggestion
  void compute_local_();
```